src/App.jsx

```diff
@@ -1,15 +1,34 @@
 import React from 'react';
 import './App.scss';
+import { Sum } from './components/Sum';
+import { SumProps } from './components/SumProps';
 
 const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove comment lines :) */}
+    <Sum
+      a={SumProps.sum1.a}
+      b={SumProps.sum1.b}
+    />
+
+    <Sum
+      a={SumProps.sum2.a}
+      b={SumProps.sum2.b}
+    />
+
+    <Sum
+      a={SumProps.sum3.a}
+      b={SumProps.sum3.b}
+    />
+
+    <Sum
+      a={SumProps.sum4.a}
+      b={SumProps.sum4.b}
+    />
+
+    <Sum
+      a={SumProps.sum5.a}
+      b={SumProps.sum5.b}
+    />
```

Comment:
I did it in a similar way but then I learned that you can simply pass the integers like this `a={2}` `b={3}` and it becomes like 10 times shorter. Just for your info.

Reply:
Okay noted thanks. Will attend to it